project/Dependencies.scala

```diff
@@ -2,5 +2,5 @@ object Dependencies {
   // scalaVersion needs to be kept in sync with travis-ci
   val Scala212 = "2.12.13"
   val Scala213 = "2.13.4"
-  val Scala3   = "3.0.0-M3"
+  val Scala3   = Seq("3.0.0-M3", "3.0.0-RC1")
```

Comment:
Why M3 as well?

Reply:
it can be convenient for downstream maintainers to publish for the last two versions, so that people can take the library version bump separately from the Scala version bump, to help narrow down the number of possible causes if something goes wrong

@dwijnand regardless, if keeping M3 makes any trouble for supporting for RC1, it is totally fine with me to just drop M3